certbot/cli.py

```diff
@@ -831,14 +831,19 @@ def prepare_and_parse_args(plugins, args, detect_defaults=False):  # pylint: dis
         metavar="DOMAIN", action=_DomainsAction, default=[],
         help="Domain names to apply. For multiple domains you can use "
              "multiple -d flags or enter a comma separated list of domains "
-             "as a parameter. (default: Ask)")
+             "as a parameter. (default: Ask) Note: the first provided domain "
+             "will be most prominent in filenames and some software user "
+             "interfaces")
```

Comment:
I think this information is inaccurate now. The first domain will not be most prominent if you use `--cert-name` or you have a matching certificate with the same names and didn't include `--duplicate`.

I also personally think the extra information after `(default: Ask)` looks bad because it breaks the convention used by all other flags, but that's a stylistic preference. If you're okay with that, we can leave it as is.